### ISSUE TITLE
Scrolling an empty CollectionView results in the app stuck with High CPU

### ISSUE THREAD

**Issue by trampster** [dotnet/maui#19770]
### Description

If you have an empty CollectionView and scroll on it for a while the CPU will get stuck at high CPU usage even after you stop scrolling.

### Steps to Reproduce

This can be reproduced using the linked repository: https://github.com/trampster/MauiEmptyCollectionCpuIssue

To reproduce:
1. Have an app with an empty CollectionView
2. Launch the app
3. Monitor CPU usage using `adb shell top`
4. Scroll on the blank screen for 10 seconds, (I find it easiest to do a drag on the screen in a circular pattern)
5. Stop scrolling and check if the CPU usage stays high, if no go back to 4

Usually after 1-2 attempts the CPU stays high (40-50%) even when not scrolling. After some time the phone gets hot.

### Link to public reproduction project repository

https://github.com/trampster/MauiEmptyCollectionCpuIssue

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Yes, this used to work in Xamarin.Forms

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Did you find any workaround?

If you add an item to the collection view then it fixes it. We have added a dummy item with IsVisible set to false to work around this.

**Comment by PureWeen** [comment]
@trampster can you try in release mode to see if you have the same issues?
https://github.com/dotnet/maui/wiki/Profiling-.NET-MAUI-Apps#feature-xyz-was-faster-in-xamarinforms

**Comment by trampster** [comment]
Yes, it also occurs in release mode although the CPU usage is divided between surface flinger and our app
* surfaceflinger 28%
* out app 20%
Please note the CPU is stuck at these levels after you stop scrolling until the app closes or the screen turns off etc.

**Comment by azrinsani** [comment]
Bump

**Comment by bebe84** [comment]
I noticed it yesterday, don't know how it is possible or this is another issue but if I add Margin="10,0" to CollectionView high CPU usage happens even without touching the screen. It severely affects performance on Android.
